.github/workflows/basic.yml

```diff
@@ -39,9 +39,12 @@ jobs:
       if: startsWith(matrix.os, 'macOS')
       shell: bash
       run: |
+        find . -name "libssl*"
```

Comment:
You probably want `find / -name "libssl*"` (or libcrypto) because otherwise you're only looking in the working directory...which won't find anything